netbench/netbench/src/multiplex.rs

```diff
@@ -510,15 +527,19 @@ mod tests {
                 let mut prev_count = 0;
                 let mut cx = core::task::Context::from_waker(&waker);
 
+                // let mut io_logger = crate::trace::StdioLogger::new(0, traces);
+
```

Comment:
can this be removed? and the other two commented out lines

Reply:
yeah. it does make it easier to debug with them in; you just remove the comments. but since things are working i guess it's less of a need.

Reply:
Wouldn't you have to change the method signature of `poll` in driver.rs as well?

Reply:
No you just comment out the existing traces

Reply:
Ah I get it now. You can leave it in if you'd like, but I'd probably put a little more instruction on it, its a little trickier to see how to use it inside a macro, my IDE at least doesn't give any suggestions on the method arguments or anything

Reply:
K. I'll try extracting the bulk of it out into a function.